ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/multipart/EditorMultiPartStackPresenter.java

```diff
@@ -138,11 +143,28 @@ public void setActivePart(@NotNull PartPresenter part) {
     }
 
     @Override
-    public void hidePart(PartPresenter part) {
-        EditorPartStack editorPartStack = getPartStackByPart(part);
-        if (editorPartStack != null) {
-            editorPartStack.hidePart(part);
-        }
+    public void maximize() {
```

Comment:
Actually the central part stack become maximized when right, left and bottom part stacks are collapsed. So, it's just to store the part stack state.